stable/jenkins/ci/agent-values.yaml

```diff
@@ -0,0 +1,9 @@
+agent:
+  yamlTemplate: |-
```

Comment:
FYI. I'd adapted this from the [k8s docs](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/). Not sure why I removed `effect` as it looks important. You could just take what's in the k8s docs. @lgg42 also provided an example [here](https://github.com/helm/charts/pull/21672#issuecomment-606662152).
```
tolerations:
- key: "key"
  operator: "Equal"
  value: "value"
  effect: "NoSchedule"
```

Reply:
I think I am going to remove these here completely. Thought about having three different values.yaml files for testing:

- one for XML configuration, all other values using defaults
- one for CasC configuration, all other values using defaults
- one other-values.yaml, which should test snippets which are not nendered within defaults

I am sure it's not possible to catch any possible combination of values and it's also not desireable as it will become a maintenance nightmare and take very long to execute, but for me it would make sense to have something in place where I could ask people to adjust the values so that it tests what there PR proposes. Or at least does not render invalid kubernetes manifests.

Reply:
Included `other-values.yaml` in #22008.